go/libraries/doltcore/sqle/reflog_table_function.go

```diff
@@ -29,10 +29,11 @@ import (
 )
 
 type ReflogTableFunction struct {
-	ctx      *sql.Context
-	database sql.Database
-	refExpr  sql.Expression
-	showAll  bool
+	ctx            *sql.Context
+	database       sql.Database
+	refAndArgExprs []sql.Expression
+	tabId          sql.TableId
+	colset         sql.ColSet
```

Comment:
These two members don't seem to be used anywhere, so should be removed.

Reply:
These were added in a separate PR, looks like they're being used a couple lines down.